net/net.sh

```diff
@@ -32,8 +32,6 @@ Operate a configured testnet
    -t edge|beta|stable|vX.Y.Z         - Deploy the latest tarball release for the
                                         specified release channel (edge|beta|stable) or release tag
                                         (vX.Y.Z)
-   -f [cargoFeatures]                 - List of |cargo --feaures=| to activate
```

Comment:
@pgarg66 - heads up that `./net.sh start -f cuda` will no longer work, it's just `./net.sh start`.   I went with removing `-f` entirely to intentionally make it difficult to add features in the future